doc/source/user_guide/groupby.rst

```diff
@@ -601,6 +601,54 @@ must be either implemented on GroupBy or available via :ref:`dispatching
    grouped.agg({'D': 'std', 'C': 'mean'})
    grouped.agg(OrderedDict([('D', 'std'), ('C', 'mean')]))
 
+.. _groupby.aggregate.keyword:
```

Comment:
I've moved this before the "Applying different functions" section. I'm not sure whether we should recommend this as the "preferred" way of doing column-specific aggregations.

On the upside, the output feels generally more useful if you're doing downstream operations, since you don't end up with a MultiIndex in the columns.

The downside is that doing multiple operations is more typing, since you need to provide a name (which is good) and repeat the column selection (not as good).

Reply:
ok that's fine, we can always adjust the docs later